server/services/yeelight/lib/features/index.js

```diff
@@ -0,0 +1,160 @@
+const { flatten, intersection, isEmpty } = require('lodash');
```

Comment:
Lodash is quite a big library, we really try to avoid installing libs just for a few functions (like "isEmpty" or flatten)

I'm sure it's doable in pure JS ( https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/flat )